### ISSUE TITLE
Minimum self-bond ratio 

### ISSUE THREAD

**Issue by gamarin2** [cosmos/cosmos-economics#14]
Self-bond ratio = (validator's self-bonded stake)/(validator's total stake)

Validators should individually choose a minimum self-bond ratio, so that they can't sneakily unbond without their delegators knowing it. If a validator's self-bond ratio goes below the predefined limit, he and all of his delegators will unbond.

This is to give delegators the certainty that their validators will always have at least a certain level of skin-in-the-game.

**Comment by rigelrozanski** [comment]
Yeah we were talking about this one a while ago. How I see it is there are two possibly enforceable parameters which validators could self-impose but would be enforced by the hub once they were defined: 
 - Minimum self bond ratio (as you've outlined)
 - Minimum self bond absolute amount
If the validator went below there pre-defined self imposed amount the hub would automatically unbond them. Sure. One annoying thing is that a minimum self bond ratio would require that a validator not accept more bonds after a certain amount if they were blowing up.
